rpcserverhandlers_test.go

```diff
@@ -2284,6 +2315,229 @@ func TestHandleGetBlockchainInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleGetBlock(t *testing.T) {
+	// Define variables related to block432100 to be used throughout the
+	// handleGetBlock tests.
+	blkHeader := block432100.Header
+	blk := dcrutil.NewBlock(&block432100)
+	blkHash := blk.Hash()
+	blkHashString := blkHash.String()
+	blkBytes, err := blk.Bytes()
+	if err != nil {
+		t.Errorf("error serializing block: %+v", err)
+	}
+	blkHexString := hex.EncodeToString(blkBytes)
+	bestHeight := int64(432151)
+	confirmations := bestHeight - blk.Height() + 1
+	nextHash := mustParseHash("000000000000000002e63055e402c823cb86c8258806508d84d6dc2a0790bd49")
+	chainWork := big.NewInt(0).SetBytes([]byte{0x0e, 0x80, 0x5f, 0xb8, 0x52,
```

Comment:
Not a big deal, but this can be simplified/compactified a bit via ` chainWork, _ := new(big.Int).SetString("0e805fb85284503581c57c", 16)`.

Reply:
Thanks, updated.